spec/src/main/resources/META-INF/smithy/bsp/tags.smithy

```diff
@@ -0,0 +1,36 @@
+$version: "2"
+
+namespace bsp
+
+// Applying the tags a-posteriori to avoid hurting the readability of the spec
+apply URI @tags(["basic"])
```

Comment:
as you can see in this file, `traits` can be applied a-posteriori : they are a little bit like Scala typeclasses in that regard. 

Because I wanted to avoid bloating the main spec file with tags, I've started tagging the shapes here in a separate file. 

The `@tags` trait could be later used to group shapes together to control how they appear in documentation.